src/glados/engine/audio_processing.py

```
@@ -0,0 +1,610 @@
+"""GLaDOS Engine Audio Processing - Audio input handling and processing pipeline.
+
+This module handles all aspects of audio input processing including voice activity
+detection, speech recognition, wake word detection, audio buffering, and playback
+management. It contains the core audio processing loop and state management for
+the voice assistant.
+"""
+
+import copy
+import queue
+import threading
+import time
```

Ruff (0.11.9)

`time` imported but unused

Remove unused import: `time`

(F401)